vyper/semantics/types/primitives.py

```diff
@@ -211,9 +211,13 @@ def _add_div_hint(node, e):
     else:
         return e
 
+    def get_source(node):
```

Comment:
maybe make this look more like a private function - `_get_source()`
```suggestion
    def _get_source(node):
```